components/link/link.js

```diff
@@ -44,13 +45,23 @@ export const linkStyles = css`
 			color: var(--d2l-color-ferrite);
 		}
 	}
+	d2l-icon.d2l-new-window {
+		color: var(--d2l-color-celestine);
+		margin-top: -7px;
```

Comment:
I have a feeling these negative margin values are working nicely for our default font size, but may start to fall apart when users choose larger/smaller ones in account settings and/or use browser zoom.

Typically a better solution would be to apply flexbox to the container (the host in this case) and use that to vertically centre everything. That's not feasible here though as it'll break everyone's stuff. You could maybe try `rem`-based negative margins instead?

Reply:
I fixed this by changing the vertical-align, setting it to inherti the text allignment worked perfectly for all the font size I tested with